configure

```diff
@@ -124,14 +125,14 @@ while :; do
 				JSON
 			)
 			if echo "$response" | grep '"token"'; then
-				echo "$response" | jq -r .token | read i_api_token
+				echo "$response" | jq -r .token | read i_access_token
 			else
 				printf "\033[31;31mError: \033[0m"
 				echo "$response" | jq -r .message
 				echo "You'll need to create a GitHub API token manually."
 			fi
 
-			[ -n "$i_api_token" ] && break
+			[ -n "$i_access_token" ] && access_token="$i_access_token" && break
```

Comment:
Rather than repeating this twice, could it be moved to the `break` in the outer look maybe? I _think_ that would let it only be set once.